src/routes/(admin)/questionbank/single-question/+page.svelte

```
@@ -0,0 +1,182 @@
+	import StateSelection from '$lib/components/StateSelection.svelte';
+	import { superForm, type Infer, type SuperValidated } from 'sveltekit-superforms';
+	import { questionSchema, type FormSchema } from './schema';
+	import { zodClient } from 'sveltekit-superforms/adapters';
+
+	const { data }: { data: { form: SuperValidated<Infer<FormSchema>> } } = $props();
+	console.log('Data:', data);
+	const {
+		form: formData,
+		enhance,
+		submit,
+		reset
```

ESLint

[error] 'reset' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)